src/sentry/models/groupassignee.py

```diff
@@ -21,24 +21,34 @@
 
 class GroupAssigneeManager(BaseManager):
     def assign(self, group, assigned_to, acting_user=None):
+        from sentry.models import User, Team
+
+        if isinstance(assigned_to, User):
```

Comment:
@tkaemming I want your opinion on this.

I originally wanted to do some abstraction of `Actor`, but turns out, this `Actor` also would need to encapsulate the same logic, and in this case `GroupAssignee` is just this actor abstraction. I also chose to do this with two different columns, `user_id` and `team_id` instead of an alternative Django "generic" FK style, where there's a column for `model` and `object_id`, this allows us to still use actual ForeignKeys, which being a tad more rigid since it can't FK against _anything_, which is a good thing here.